examples/starter-policy/fastapi/sample_application.py

```diff
@@ -0,0 +1,176 @@
+import oso_sdk
+from fastapi import Depends, FastAPI, Request
+from oso_sdk.integrations.fastapi import FastApiIntegration
+
+# remember to update this before running the server!
+API_KEY = "<please provide your api key here>"
+
+oso = oso_sdk.init(
+    API_KEY,
+    FastApiIntegration(),
+    # create a local instance of the Oso SDK
+    shared=False,
+    # only enforce routes with the `@oso.enforce` decorator
+    optin=True,
+)
+
+app = FastAPI(dependencies=[Depends(oso)])
+
+
+@oso.identify_user_from_request
+async def user(request: Request) -> str:
+    """
+    Oso Cloud handles authorization - you'll need to handle your own
+    authentication. That means it's up to you to figure out *who* a user is
+    (for example, with a username and password).
+
+    For this sample application, we identify every user as "anonymous".
+
+    This means that the actor for requests is identified as `User{"anonymous"}`.
+    """
+    return "anonymous"
+
+
+@app.get("/org/{id}")
+@oso.enforce(
+    # actor
+    "{id}",
+    # action
+    "view",
+    # resource
+    "Organization",
+)
+async def get_organization(id: str):
+    """
+    This is the "view" endpoint for an Organization, keyed on `id`.
+
+    When this endpoint is accessed, we check to see if the actor has "view"
+    permissions on the Organization in question.
+
+    If you'd like to give the anonymous user access to GET `/org/acme`, you
+    can add this fact to Oso Cloud:
+
+        has_role(User{"anonymous"}, "viewer", Organization{"acme"})
+
+    Because the "viewer" role gives "view" permission, this will allow access.
+    You can also add a fact giving this permission directly:
+
+        has_permission(User{"anonymous"}, "view", Organization{"acme"})
```

Comment:
Similar comment to the one on the README, we should only provide one option in these cases. There are other resources to dig into alternatives.